components/kyma-environment-broker/internal/broker/instance_create_test.go

```diff
@@ -530,22 +530,17 @@ func TestProvision_Provision(t *testing.T) {
 }
 
 func fixExistOperation() internal.ProvisioningOperation {
-	return internal.ProvisioningOperation{
-		Operation: internal.Operation{
-			ID:         existOperationID,
-			InstanceID: instanceID,
-			ProvisioningParameters: internal.ProvisioningParameters{
-				PlanID:    planID,
-				ServiceID: serviceID,
-				ErsContext: internal.ERSContext{
-					SubAccountID:    subAccountID,
-					GlobalAccountID: globalAccountID,
-				},
-				Parameters:     internal.ProvisioningParametersDTO{Name: clusterName},
-				PlatformRegion: region,
-			},
-		},
-	}
+	provisioningOperation := fixture.FixProvisioningOperation(existOperationID, instanceID)
+	pp := internal.ProvisioningParameters{}
+	pp.PlanID = planID
+	pp.ServiceID = serviceID
+	pp.ErsContext.SubAccountID = subAccountID
+	pp.ErsContext.GlobalAccountID = globalAccountID
+	pp.Parameters.Name = clusterName
+	pp.PlatformRegion = region
+	provisioningOperation.ProvisioningParameters = pp
```

Comment:
nit:
as we're replacing whole `ProvisioningParameters` struct field of `ProvisioningOperation`, we can do it like this to avoid `pp.*` lines:

```suggestion
	provisioningOperation.ProvisioningParameters = internal.ProvisioningParameters{
		PlanID:        planID,
		ServiceID:      serviceID,
		ErsContext:     internal.ERSContext{
			SubAccountID:    subAccountID,
			GlobalAccountID: globalAccountID,
		},
		Parameters:     internal.ProvisioningParametersDTO{
			Name:                        clusterName,
		},
		PlatformRegion: region,
	}
```

Reply:
Done.